packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -275,6 +283,18 @@ export class AtomicSearchInterface {
     this.engine.executeFirstSearchAfterStandaloneSearchBoxRedirect(analytics);
   }
 
+  private engineIsCreated(engine?: SearchEngine): engine is SearchEngine {
+    if (!engine) {
+      console.error(
```

Comment:
This error is much more clear than the previous Headless undefined engine error 😄